datadog-agent.advisories.yaml

```diff
@@ -497,6 +497,10 @@ advisories:
             componentType: python
             componentLocation: /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/METADATA, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/RECORD, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/direct_url.json, /usr/share/datadog-agent/lib/python3.11/site-packages/snowflake_connector_python-3.12.1.dist-info/top_level.txt
             scanner: grype
+      - timestamp: 2024-11-06T11:09:36Z
+        type: pending-upstream-fix
+        data:
+          note: Upstream package repository doesn't contain yet snowflake_connector_python-3.12.3. See https://agent-int-packages.datadoghq.com/external/snowflake-connector-python/.
```

Comment:
Do you mean it is not available as part of their hosted datadog dependencies ? If so, we might want to add that on the note.

Reply:
Yes exactly. Thanks, I'm going to make it more clear

Reply:
@hectorj2f PTAL. Is that more clear now? Thanks

Reply:
Thanks @maxgio92